test/integration/cve_int_test.go

```diff
@@ -120,7 +120,7 @@ func (suite *CveIntegrationTestSuite) TestJSON() {
 	cp.Expect(`"project":`)
 	cp.Expect(`"commitID":`)
 	cp.ExpectExitCode(0)
-	AssertValidJSON(suite.T(), cp)
+	// AssertValidJSON(suite.T(), cp) // report is too large to fit in terminal snapshot
```

Comment:
This _seems_ like a valuable thing to test. Is there any plan to address this or should this line be removed completely?

Reply:
These lines are sprinkled throughout the integration test suite. There is a desire to eventually test actual JSON content, but that is a ways down the road. The fact that we're using Go's unmarshaling for JSON output, and the fact that there are other `AssertValidJSON()` calls in unit tests, brings us comfort.